whisper/src/bin/whisper-update.rs

```diff
@@ -0,0 +1,32 @@
+#[macro_use]
+extern crate structopt;
+extern crate failure;
+extern crate whisper;
+
+use failure::Error;
+use std::path::PathBuf;
+use std::time::{SystemTime, UNIX_EPOCH};
+use structopt::StructOpt;
+use whisper::point::Point;
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "whisper-update")]
+struct Args {
+    /// Path to data file
+    #[structopt(name = "path", parse(from_os_str))]
+    path: PathBuf,
+
+    /// Set of data points
+    #[structopt(name = "timestamp:value")]
+    points: Vec<Point>,
+}
+
+fn main() -> Result<(), Error> {
+    let args = Args::from_args();
+    println!("whisper-update {}", env!("CARGO_PKG_VERSION"));
+    println!("{:?}", args);
+
+    let now = SystemTime::now().duration_since(UNIX_EPOCH)?.as_secs() as u32;
```

Comment:
```
if len(datapoints) == 1:
   timestamp, value = datapoints[0]
   whisper.update(path, value, timestamp)
 else:
   whisper.update_many(path, datapoints)
```

this original (python) logic is missing

Reply:
It was already discussed.  It should not copy the original logic 1x1.

Reply:
Any reason for breaking compatibility with existing (python) cli tools?
I thought the idea is to provide drop-in replacements for these

Reply:
I'm not agains the new behaviour, but we might choose some new bin name not to confuse the users

Reply:
The only issue I see here is possible performance penalty.

Reply:
> I'm not agains the new behaviour, but we might choose some new bin name not to confuse the users

Behaviour is the same. The difference in custom performance optimisations, which is not very valuable for cli implementation. Also Rust compiler can level down this custom optimisation.